server/configs/config.json

```diff
@@ -6,13 +6,10 @@
     "dialect": "postgres",
     "options": {
       "host": "127.0.0.1",
-      "dialect": "postgres",
-      "operatorsAliases": {
-        "$ne": "Op.ne"
-      }
```

Comment:
Why does this need to be removed?

Reply:
String based operators are now deprecated in sequelize and we don't actually use it.